### PR TITLE
Update build

### PR DESCRIPTION
**Pull request by SuperQ**
* Update to Ruby 2.6.x.
* Bump Gemfile vendor locking.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM   ruby:2.4-onbuild
+FROM   ruby:2.7-onbuild
```
Comment (roidelapluie) [inline Dockerfile:1]: That image does not exist

**Comment by roidelapluie** [comment]
That probably does not pass docker build.

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
-CMD    bundle exec nanoc view --host 0.0.0.0
-VOLUME output
+FROM    ruby:2.7
+EXPOSE  3000
+WORKDIR /usr/src/app
+COPY    Gemfile /usr/src/app/
+COPY    Gemfile.lock /usr/src/app/
+RUN     bundle install
+COPY    . /usr/src/app
+RUN     bundle exec nanoc
```
Comment (roidelapluie) [inline Dockerfile:8]:
```suggestion
RUN     bundle exec nanoc --verbose
```

**Comment by roidelapluie** [comment]
https://github.com/nanoc/nanoc/issues/1501 -> I'd recommend to stay at 2.6

**Comment by SuperQ** [comment]
Ok, I rolled it back to 2.6.x

**Comment by ddfreyne** [comment]
Ruby 2.7 is now usable as of Nanoc 4.11.19. The underlying issue (https://github.com/nanoc/nanoc/issues/1501) is resolved.

**Pull request merged**